src/tests/base.cpp

```diff
@@ -19,6 +22,29 @@ void TestBase::storeConfig(ConfigNode& config)
 
 TestBase::TestBase(const Context& context) : _context(context) {}
 
+std::string TestBase::extractFilename(const std::string& path)
+{
+    auto pos = path.find_last_of('/');
```

Comment:
I guess we should handle windows as well: '\\\\'

Reply:
Fixed in 6af65e4.